04-store-api/starter/QuizAnswers2.txt

```diff
@@ -0,0 +1,21 @@
+1. In this lesson, you created a middleware function called asyncWrapper. Why?
+The function was created to avoid repeating the same try-catch error handling code in multiple asynchronous route handlers. 
```

Comment:
Can we use `try-catch` to handle exceptions when we call an async function from a sync function?

Reply:
No, we cannot use ```try-catch``` to handle exceptions when calling an async function from a sync function. Instead, we should use ```.catch()``` on the Promise returned by the async function or use an async function with await to handle exceptions because the error is thrown asynchronously and outside the synchronous call stack, hence it cannot be caught by the try-catch block in the sync function.